test/index.js

```diff
@@ -178,7 +179,7 @@ test('Textbox:help', function (tape) {
 });
 
 test('Textbox:value', function (tape) {
-  tape.plan(3);
+  tape.plan(6);
```

Comment:
Please add a comment explaining what this magic number does or turn it into a variable that explains it.

Reply:
The number is just to specify how many assertions we are going to run in the test. tape has `.plan` at the beginning of every test.

Reply:
[Is it strictly necessary?}(https://github.com/substack/tape#tplann) The docs don't make it clear at a glance. Looks like there's a conflict to be resolved in the PR.

Reply:
Yea, it is. If we don't declare the plan, it will error out. 

I resolved the conflict. One question, I saw that we removed all the rollup packages on master. Liv mentioned below that the build command for this package uses rollup. I assume that we made some changes?

Reply:
Yes, I reverted those non-working changes as they were in Carolyn's way. Should be fine to leave them out now.

Reply:
Oh ok, makes sense.